lib/tests/systems.nix

```diff
@@ -14,10 +14,10 @@ let mseteq = x: y: lib.sort lib.lessThan x == lib.sort lib.lessThan y; in
 {
   all = assertTrue (mseteq all (linux ++ darwin ++ cygwin ++ freebsd ++ openbsd ++ netbsd ++ illumos));
 
-  arm = assertTrue (mseteq arm [ "armv5tel-linux" "armv6l-linux" "armv7l-linux" ]);
-  i686 = assertTrue (mseteq i686 [ "i686-linux" "i686-freebsd" "i686-netbsd" "i686-cygwin" ]);
+  arm = assertTrue (mseteq arm [ "armv5tel-linux" "armv6l-linux" "armv7l-linux" "aarch64-linux" ]);
```

Comment:
I think there was a commit, which removed explicitly aarch64 from arm, because the cpu architecture is different.